src/PrestaShopBundle/Resources/views/Admin/Module/catalog.html.twig

```diff
@@ -30,6 +30,14 @@
             {% if topMenuData.categories is defined %}
                 {% include 'PrestaShopBundle:Admin/Module/Includes:categories_grid.html.twig' with { 'categories' : topMenuData.categories }  %}
             {% endif %}
+            <script>
```

Comment:
ping @nihco2 I need help, even when I pray gods, I can't display a poppin when poppin content is retrieved using ajax request :/